sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/planning/SelectedFieldSuite.scala

```diff
@@ -0,0 +1,440 @@
+        StructField("field6", StructType(
+          StructField("subfield1", StringType, nullable = false) ::
+          StructField("subfield2", StringType) :: Nil)) ::
+        StructField("field7", StructType(
+          StructField("subfield1", StructType(
+            StructField("subsubfield1", IntegerType) ::
+            StructField("subsubfield2", IntegerType) :: Nil)) :: Nil)) ::
+        StructField("field8", MapType(StringType, ArrayType(StructType(
+          StructField("subfield1", IntegerType) ::
+          StructField("subfield2", ArrayType(IntegerType, containsNull = false)) :: Nil)), valueContainsNull = false)) ::
+        StructField("field9", MapType(StringType, IntegerType, valueContainsNull = false)) :: Nil)) ::
+     StructField("col3", ArrayType(StructType(
+       StructField("field1", StructType(
+         StructField("subfield1", IntegerType, nullable = false) ::
+         StructField("subfield2", IntegerType) :: Nil)) ::
+       StructField("field2", MapType(StringType, IntegerType, valueContainsNull = false)) :: Nil), containsNull = false), nullable = false) ::
+     StructField("col4", MapType(StringType, StructType(
+       StructField("field1", StructType(
+         StructField("subfield1", IntegerType, nullable = false) ::
+         StructField("subfield2", IntegerType) :: Nil)) ::
+       StructField("field2", MapType(StringType, IntegerType, valueContainsNull = false)) :: Nil), valueContainsNull = false), nullable = false) ::
+     StructField("col5", ArrayType(MapType(StringType, StructType(
+       StructField("field1", StructType(
+         StructField("subfield1", IntegerType) ::
+         StructField("subfield2", IntegerType) :: Nil)) :: Nil), valueContainsNull = false))) ::
+     StructField("col6", MapType(StringType, ArrayType(StructType(
+       StructField("field1", StructType(
+         StructField("subfield1", IntegerType) ::
+         StructField("subfield2", IntegerType) :: Nil)) :: Nil), containsNull = false))) ::
+     StructField("col7", ArrayType(StructType(
+       StructField("field1", IntegerType, nullable = false) ::
+       StructField("field2", StructType(
+         StructField("subfield1", IntegerType, nullable = false) :: Nil)) ::
+       StructField("field3", ArrayType(StructType(
+         StructField("subfield1", IntegerType, nullable = false) :: Nil))) :: Nil))) ::
+     StructField("col8", ArrayType(StructType(
+       StructField("field1", ArrayType(IntegerType, containsNull = false), nullable = false) :: Nil))) :: Nil)
+
+  private val testRelation = LocalRelation(schema.toAttributes)
+
+  override def beforeAll() {
+    val indentedSchema = schema.treeString.split('\n').map(x => "  " + x).mkString("\n") + "\n"
+    // Print the test schema at the beginning of the test
+    // scalastyle:off println
+    println("Testing SelectedField extractor using the following schema:\n" + indentedSchema)
+    // scalastyle:on println
```

Comment:
I think we can get rid of this print now? Do we want to print it in every test?

Reply:
Yes, I suppose it's enough to embed the tree string as a comment above, which was basically a copy-n-paste of the `println`. I'll get rid of this.